_includes/main.scss

```diff
@@ -229,7 +237,7 @@ ul {
 
 .d\:c-1\/3 {
   @include desktop {
-    flex: 1 1 calc(33.33% - 1rem);
+    flex: 0 1 calc(33.33% - 1rem);
```

Comment:
this just forces the columns in the footer to always be 1/3 on desktop

the collective actions link that was added took up 2/3s which causes it to be out of alignment